src/routes/goals/index.js

```diff
@@ -14,5 +15,6 @@ router.get('/', transactionWrapper(retrieveGoalsByIds));
 router.get('/:goalId/recipient/:recipientId', transactionWrapper(retrieveGoalByIdAndRecipient));
 router.put('/changeStatus', transactionWrapper(changeGoalStatus));
 router.delete('/', transactionWrapper(deleteGoal));
+router.post('/similar', transactionWrapper(postSimilarGoalsForRecipient));
```

Comment:
any reason why this is a post and not a get?

Reply:
oversight on my part, made it a get here: 66e2a0450bd2a72cff78d4b2ae89d238342ee1b2